internal/gitlab/codequality.go

```diff
@@ -46,17 +54,63 @@ func NewCodeQuality(diff yamlfmt.FileDiff) (CodeQuality, bool) {
 		return CodeQuality{}, false
 	}
 
+	begin, end := detectChangedLines(&diff)
+
 	return CodeQuality{
 		Description: "Not formatted correctly, run yamlfmt to resolve.",
 		Name:        "yamlfmt",
 		Fingerprint: fingerprint(diff),
 		Severity:    Major,
 		Location: Location{
 			Path: diff.Path,
+			Lines: &Lines{
+				Begin: begin,
+				End:   end,
+			},
 		},
 	}, true
 }
 
+// detectChangedLines finds the first and last lines that differ between original and formatted content.
+func detectChangedLines(diff *yamlfmt.FileDiff) (begin int, end *int) {
+	original := strings.Split(string(diff.Diff.Original), "\n")
+	formatted := strings.Split(string(diff.Diff.Formatted), "\n")
+
+	max := len(original)
```

Comment:
This can just use `math.Max` instead.

Reply:
Right, perhaps we can use the [`max` builtin`](https://pkg.go.dev/builtin#max) then.